### PR TITLE
Integration specs

### PR DESCRIPTION
**Pull request by shahira-sadat**
- Use Capybara to write integration tests for each view in your project.
- 
- User index page:
  - I can see the username of all other users.
  - I can see the profile picture for each user.
  - I can see the number of posts each user has written.
  - When I click on a user, I am redirected to that user's show page.
  - user show page:

  - I can see the user's profile picture.
  - I can see the user's username.
  - I can see the number of posts the user has written.
  - I can see the user's bio.
  - I can see the user's first 3 posts.
  - I can see a button that lets me view all of a user's posts.
  - When I click a user's post, it redirects me to that post's show page.
  - When I click to see all posts, it redirects me to the user's post's index page.
  
- User post index page:
  - I can see the user's profile picture.
  - I can see the user's username.
  - I can see the number of posts the user has written.
  - I can see a post's title.
  - I can see some of the post's body.
  - I can see the first comments on a post.
  - I can see how many comments a post has.
  - I can see how many likes a post has.
  - I can see a section for pagination if there are more posts than fit on the view.
  - When I click on a post, it redirects me to that post's show page.
  
- Post show page:
  - I can see the post's title.
  - I can see who wrote the post.
  - I can see how many comments it has.
  - I can see how many likes it has.
  - I can see the post body.
  - I can see the username of each commenter.
  - I can see the comment each commentor left.

### CODE REVIEW

File: spec/author_spec.rb [hunk 1]
```diff
@@ -1,49 +1,29 @@
...
+        subject.id = 1
+        Post.create(title: 'title', text: "test post #{number}", author: subject)
       end
     end
 
-    it 'should return the 3 latest posts' do
-      expect(subject.three_recent_posts_per_user).to eq(Post.order(created_at: :desc).limit(3))
-      expect(subject.three_recent_posts_per_user).to eq(Post.last(3))
+    it 'loads only the first three posts' do
+      expect(subject.three_recent_posts_per_user.length).to eq(3)
```
Comment (DeliceLydia) [inline spec/author_spec.rb:26]: - [ ] Please fix tests that are failing
Screenshot from 2022-11-01 19-53-02

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @shahira-sadat, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: spec/author_spec.rb (hunk 1 above)
Comment (shahira-sadat) [reply spec/author_spec.rb:26]: Screen Shot 2022-11-01 at 10 41 18 PM

@DeliceLydia 
On my PC, its working and has no problem. what should I do now??

Comment (DeliceLydia) [reply spec/author_spec.rb:26]: @shahira-sadat ooh really? request another review may be another reviewer will not have a problem
and you can share a screenshot

File: spec/comment_spec.rb [hunk 2]
```diff
@@ -1,25 +1,31 @@
 require 'rails_helper'
```
Comment (tresorsawasawa) [inline spec/comment_spec.rb:2]: :clap: :clap: :clap: , all your tests are passing :100: :+1: Good job
Screenshot from 2022-11-02 01-11-34

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @shahira-sadat 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**